ui/partials/main/thWatchedRepoNavPanel.html

```diff
@@ -4,6 +4,13 @@
     <span>
       <form role="search" class="form-inline">
 
+        <span class="btn btn-sm btn-view-nav nav-menu-btn" 
+              ng-show="serverChanged" ng-cloak
+              ng-click="updateButtonClick()"
+              id="revisionChangedLabel"
+              title="New version of Treeherder has been deployed. Reload to pick up changes.">
+          <span class="fa fa-exclamation-circle" />&nbsp;Treeherder Update Available
```

Comment:
Looks a bit strange to have this capitalized, let's just say "Treeherder update available"